### DISCUSSION TITLE
Eric Cieszynski - Introduction

### DISCUSSION THREAD

**Discussion by ecieszynski** [cooperrc/computational-mechanics#16] [General]
Hello, my name is Eric Cieszynski and I am entering my Senior year here at UConn, and am a Mechanical Engineering major with aerospace focus. I am here to get ahead on my last year by taking this class over the summer. This class is perfect for me to take right now as I begin my internship at Brooks Automation, because it requires me to know python. What I hope this course can go over are machine learning applications as well as some aerospace problems to help me learn more in those specific areas.

**Comment by tempcooper** [comment]
> This class is perfect for me to take right now as I begin my internship at Brooks Automation

Amazing timing! If you run into issues from your internship (that you can safely share), I'd love to see how you're using Python to automate your processes. I'm happy to help troubleshoot. 

> What I hope this course can go over are machine learning applications as well as some aerospace problems to help me learn more in those specific areas.

> Machine learning

I _love_ machine learning applications, but they go a little outside the scope of this course. We will cover the intro-level machine learning concepts: 
- [Linear Algebra](https://cooperrc.github.io/computational-mechanics/module_04/01_Linear-Algebra.html)
- [Linear Regression](https://cooperrc.github.io/computational-mechanics/module_04/03_Linear-regression-algebra.html)

Machine learning takes these two concepts a little further into _nonlinear_ functions e.g. [Deep learning on MNIST numerals via NumPy](https://numpy.org/numpy-tutorials/content/tutorial-deep-learning-on-mnist.html)
